python/dgl/nn/pytorch/conv/sageconv.py

```diff
@@ -92,9 +98,11 @@ def forward(self, graph, feat):
         ----------
         graph : DGLGraph
             The graph.
-        feat : torch.Tensor
-            The input feature of shape :math:`(N, D_{in})` where :math:`D_{in}`
-            is size of input feature, :math:`N` is the number of nodes.
+        feat : torch.Tensor or pair of torch.Tensor
+            If a torch.Tensor is given, the input feature of shape :math:`(N, D_{in})` where
+            :math:`D_{in}` is size of input feature, :math:`N` is the number of nodes.
+            If a pair of torch.Tensor is given, the pair must contain two tensors of shape
+            :math:`(N_{in}, D_{in})` and :math:`(N_{out}, D_{in})`.
```

Comment:
They *should* be in different shapes in case of bipartite graphs.

The second `D_{in}` is a typo; it should be something like `D_{in_{dst}}`.  Will fix.

Reply:
> The second `D_{in}` is a typo; it should be something like `D_{in_{dst}}`. Will fix.

Yeah, that's what I mean.